oshi-core/src/main/java/oshi/hardware/common/AbstractNetworkIF.java

```diff
@@ -229,6 +229,12 @@ public boolean isConnectorPresent() {
         return false;
     }
 
+    @Override
+    public String getAlias() {
+        // default
+        return "";
```

Comment:
Is there a default parsing of `ifconfig` we can put in here that's reasonably consistent on *nix machines?